src/components/ViewerTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+import { ModelInfo } from '../constants';
+
+import '../../css/viewer_title.css';
+interface ViewerTitleProps {
+  modelInfo: ModelInfo;
+}
+
+const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
+  props: ViewerTitleProps
+): JSX.Element => {
+  const hasMetaData = (): boolean => {
+    for (const key in props.modelInfo) {
+      if (
+        key !== 'title' &&
+        props.modelInfo[key as keyof ModelInfo] !== undefined
+      ) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  return (
+    <div className="title-container">
+      <div className="title">
+        {props.modelInfo.title}
```

Comment:
Should there be error handling here for if `props.modelInfo.title` is undefined? Since it looks like it's an optional property in `ModelInfo`.

Reply:
Maybe a better question is whether the component should still render, and if it should show the additional metadata in the info button, if there's no defined title.

Reply:
![image](https://github.com/simularium/nbsv/assets/30200665/2442c447-2c22-4d78-a0bc-2a4a998de0e9)

Here's what it looks like if no title is defined, also:

Reply:
In theory you could have a trajectory with no title, but with some other meta data, like author names. That case wasn't specified in the designs. My assumption is that we would still want to render the component to make the info panel accessible.

So yes, we could add error handling, maybe if we wanted to adjust the placement of the icon in a case where there is no title provided, but also the default view in your screenshot is an option. I will ask @TravisKroeker. What do you think @ShrimpCryptid, does it look weird having the Icon dead center with no title?

Reply:
I kind of like having a placeholder, like "*Untitled Model*" or trajectory or something?

Reply:
Good idea! Sorry lost this suggestion in the shuffle and Megan just reminded me.